### PR TITLE
Tests for implemented interpreter ops

### PR DESCRIPTION
**Pull request by Fly-Style**
Also, added GH actions for unit & integration tests

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -5,7 +5,7 @@ use std::collections::BTreeMap;
 
 mod bytecode;
 mod interpreter;
-
+mod interpreter_test;
```
Comment (pluralia) [inline src/main.rs:8]: A blank line?

File: src/interpreter_test.rs [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+        assert_eq!(interpreter.pc, 5);
+    }
+
+    /// The interpreter stores '5' to local from the stack and
+    /// then return '5' as address from stack frame.
+    #[test]
+    fn test_store() {
+        let mut instr: BTreeMap<i32, Bytecode> = BTreeMap::new();
+        instr.insert(0, Bytecode::new(Opcode::Push, vec![5; 1]));
```
Comment (pluralia) [inline src/interpreter_test.rs:65]: Why 2 operands? Looks like work for verifier!

**Review by pluralia — APPROVED** [review]
1. There are a couple of warnings from the compiler -- do we decide to ignore them for now?
2. You were going to write a macro to make adding instructions easier. Is the idea on hold or dropped?
3. Current tests rely on `pc` -- should other types of tests be added? For example, check that `stack` / `frame` have no side effects after running.

File: src/interpreter_test.rs (hunk 2 above)
Comment (Fly-Style) [reply src/interpreter_test.rs:65]: > Why 2 operands?

`vec![5; 1]` creates vector with single element **5**.

File: src/main.rs (hunk 1 above)
Comment (Fly-Style) [reply src/main.rs:8]: Linter doesn't yell :)

**Comment by Fly-Style** [comment]
> There are a couple of warnings from the compiler -- do we decide to ignore them for now?

Yes. Most of them are unused code, but we want to preserve it as a ... let's say, carcass.

> You were going to write a macro to make adding instructions easier. Is the idea on hold or dropped?

Just the consequent idea to implement :) 

> Current tests rely on pc -- should other types of tests be added? For example, check that stack / frame have no side effects after running.

It's an interesting story: I tried to make them more 'deeper', but borrow checker won that day. :(

**Pull request merged**